runtime/dart_isolate.h

```diff
@@ -425,7 +428,8 @@ class DartIsolate : public UIDartState {
 
   DartIsolate(const Settings& settings,
               bool is_root_isolate,
-              const UIDartState::Context& context);
+              const UIDartState::Context& context,
+              bool spawn_in_group = false);
```

Comment:
I think this should be named more apropos to its semantics, like "should_load_kernel_snapshot_".  It doesn't really control how it relates to being in a group.

Reply:
The isolate may or may not be running from kernel though. This is asking whether the isolate is being spawned in a group. Maybe like `is_spawning_in_group`?

Reply:
Yea that's better.  It still seems like what we are interested is a derivative of that but I'll defer to your judgement =)